picard/ui/aboutdialog.py

```diff
@@ -67,7 +66,7 @@ def load(self):
         else:
             args["translator_credits"] = ""
 
-        args['third_parties_versions'] = '<br />'.join(["%s %s" %
+        args['third_parties_versions'] = ', '.join(["%s %s" %
```

Comment:
Actually we could replace spaces in version strings by non-breaking spaces so it wraps more nicely.

```python
        args['third_parties_versions'] = ', '.join(
            map(lambda s: s.replace(' ', '&nbsp;'),
                ["%s %s" % (versions.version_name(name), value) for name, value in versions.as_dict(i18n=True).items()
                 if name != 'version']
                )
        )
```

With approximately the same width, without:

![image](https://user-images.githubusercontent.com/151042/85469308-c9eddc80-b5ad-11ea-93a7-c6cb757f9cd2.png)

With this change:

![image](https://user-images.githubusercontent.com/151042/85469436-f144a980-b5ad-11ea-85f8-091bee9bb147.png)

Reply:
Good idea. I coded it a bit differently and also replace `-` with a non-breaking hyphen. Otherwise python-libdiscid could break like

> Discid discid 1.1.0 (compat layer from python‑
> libdiscid 1.1)